<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IToRouteNameProjectDialog</class>
 <widget class="QDialog" name="IToRouteNameProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert to Route</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRoutName">
     <property name="text">
      <string>Route name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditRouteName"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBoxSubPoints">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include sub-points of tracks in converted route.&lt;br/&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Enabling this feature leads to routes with many points. Not recommendet for devices which have only limited space for saving route points.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelSelList">
     <property name="text">
      <string>Select project from list or enter new project name.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidgetProjects"/>
   </item>
   <item>
    <widget class="QLabel" name="labelNewProjName">
     <property name="text">
      <string>New project's name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditNewProject"/>
   </item>
   <item>
    <widget class="QLabel" name="labelCreatedAs">
     <property name="text">
      <string>New project is created as:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelProjType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioQms">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>*.qms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioGpx">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>*.gpx</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDatabase">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IToRouteNameProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IToRouteNameProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
